Plugin.php

```diff
@@ -46,6 +47,10 @@ public function boot()
                 app('sentry')->captureException($exception);
             }
         });
+
+        if ($jsDsn = env('SENTRY_JAVASCRIPT_DSN')) {
```

Comment:
Use the config instead here, `sentry.dsn_js`, set in `winter.sentry.packages.sentry/sentry-laravel.config.dsn_js`. That way it can be dynamically configured on a per request basis, for instance if someone wanted to inject a different DSN in the backend vs the frontend contexts.

Reply:
I did try this before, but ran into this issue:
![image](https://user-images.githubusercontent.com/31214002/218085135-25ce3a45-755c-401a-8d45-c9be892980c3.png)

Reply:
@jaxwilko that's because you're using `js_dsn`, it's defined as `dsn_js`

Reply:
Yeah but it doesn't matter what it's called, if you add something that is missing from the predefined list of options, it throws the exception, see:
![image](https://user-images.githubusercontent.com/31214002/221588635-35d8882c-6624-4ff2-be01-bd443127fff8.png)
![image](https://user-images.githubusercontent.com/31214002/221588725-a07e40dc-35ce-45ce-a793-4c271973f131.png)
![image](https://user-images.githubusercontent.com/31214002/221588743-233b8995-d9fd-4e41-8794-83bb2def6947.png)